api/python/quilt3/packages.py

```diff
@@ -718,7 +754,31 @@ def get(self, logical_key):
             raise ValueError("Key does not point to a PackageEntry")
         return obj.get()
 
-    # def get_as_pathlib(self):
+
+
+
+    def readme(self):
+        """
+        Find the README PackageEntry. Returns (readme_logical_key, readme_package_entry)
+
+        The README is the entry with the logical key 'readme.md' (case-insensitive). Will raise a QuiltException if
```

Comment:
Do we really want it to be case-insensitive? I wouldn't bother with the extra complexity.